microsoftgraph/microsoft-graph-explorer-v4
# Task: Add support for persisting collections

## Overview
Closes #2408 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Resources tab
Click on the menu icon then 'Add to collection'
Reload the page
Notice that the collection is persisted between reloads
Edit the collection and reload the page
Notice that all edits are updated on the stored collection
